### PR TITLE
Hotfix TOC Documentation Operation.md

### LINKED ISSUE #44: BUG on table of content with github-pages / Operation Part
### Describe the bug

When you make a markdown hierachisation for the table of content, it doesn't match with the build documentation

### To Reproduce

> **note**: You might want to use the Vagrant environment to create a reproducible scenario triggering the bug.

1. 'Go on https://thefkincompany.github.io/allspark/operation/'
2. 'Look for the table of content'
3. 'If you update the title with the good tree it doesn't update the table of content according to the documentation'

### Expected behavior

If i make a table of content with h1title / h2title / h1title / h2title
I normaly have a table of content with 
h1title
h2title
h1title
h2title

### Allspark informations

- **Allspark**
  - *configuration*
  ```yaml
  # allspark group_vars/* configurations
  ```
- **Control machine**
  - *Operating system* : '...
  - *Ansible version* :
  ```
  output of `ansible --version`
  ```
  - *Vagrant version* (if used to reproduce the issue) : '...'
- **Allspark server**
  - *Operating system* :
  - *Docker version* :

### Additional context

Add any other context about the problem here. (e.g: logs)
image

### PR DESCRIPTION
**Pull request by guiadco**
### Current behaviour
The Table of content is not correctly generated
### Expected behaviour
Have a beautiful and correct TOC
### Changes proposed

-  [ ] Begin the title at ##, not a single one because it is for the page title in that case "Operation"

### Related issues

- #44 

> **note** : Put closes #44 in your commit message to auto-close the issue that your PR fixes (if such).

### CODE REVIEW

File: docs/pages/operation.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-## Requirements
+## Backup guide
+### Requirements
 
-### On the allspark machine
+#### On the allspark machine
 - Have an allspark instance running
 
-## Ansible Backup playbook
+#### Ansible Backup playbook
```
Comment (Xide) [inline docs/pages/operation.md]: Should be aligned with Requirements, not with "on the allspark machine". (one # less)

File: docs/pages/operation.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ ansible-playbook -i hosts backup.yml
     - Previous datas of the restored containers will be lost (and aligned on the backup ones)
 
 
-## Ansible Restore playbook
+#### Ansible Restore playbook
```
Comment (Xide) [inline docs/pages/operation.md]: Should be aligned with Requirements, not with "on the allspark machine". (one # less)

**Comment by guiadco** [comment]
You can check and merge if it's ok.

**Comment by Xide** [comment]
lgtm

**Pull request merged**